core/src/jvmMain/kotlin/info/laht/threekt/Window.kt

```diff
@@ -77,14 +77,21 @@ class Window @JvmOverloads constructor(
         glfwSetKeyCallback(hwnd) { _, key, _, action, _ ->
             if (key == GLFW_KEY_ESCAPE && action == GLFW_RELEASE) {
                 glfwSetWindowShouldClose(hwnd, true) // We will detect this in the rendering loop
-            } else {
-                keyListeners?.also { listeners ->
-                    val evt = KeyEvent(key, action.toKeyAction())
-                    listeners.forEach {
-                        it.onKeyPressed(evt)
-                    }
-                }
-            }
+            } else if (action == GLFW_RELEASE) {
```

Comment:
KeyEvent has a `action` property. Either we rename `onKeyPress` to `onKeyEvent` or remove the action field and add functions for `onKeyPressed`, `onKeyReleased` and `onKeyRepeat`

Reply:
Uhm... I think it would be more suitable to get rid of the `action` property and keep the `onKeyPressed`, `onKeyReleased` and `onKeyRepeat` functions. Like Oracle did in Java 8: https://docs.oracle.com/javase/8/docs/api/java/awt/event/KeyListener.html

What do you think?

Reply:
I think I prefer to remove action and use callbacks for each type